packages/pluggableWidgets/accordion-web/src/components/Header.tsx

```diff
@@ -0,0 +1,33 @@
+import { createElement, ReactElement } from "react";
+
+export interface HeaderProps {
+    heading: "headingOne" | "headingTwo" | "headingThree" | "headingFour" | "headingFive" | "headingSix";
+    text?: string;
+}
+
+export function Header(props: HeaderProps): ReactElement {
+    let header;
+
+    switch (props.heading) {
+        case "headingOne":
+            header = <h1>{props.text}</h1>;
```

Comment:
We can instead have `let HeaderComponent = <h1 />` and all the other cases still in the switch and then in the return say `<HeaderComponent>{props.text}</HeaderComponent>`

Reply:
Changed it.